packages/app/src/webapp/cli.ts

```diff
@@ -137,9 +137,7 @@ export const pasteQueuedInput = (value: string) => {
 const handleQueuedInput = async (nextBlock: HTMLElement) => {
   const queuedInput = disableInputQueueing()
 
-  if (!queuedInput) {
-    return
-  } else if (nextBlock && queuedInput.length > 0) {
+  if (nextBlock && queuedInput.length > 0) {
```

Comment:
Thanks for updating! We may still need the guard here. What about ```if(nextBlock && queuedInput && queuedInput.length > 0)```

Reply:
Right!